sal_interfaces/HVAC/HVAC_Events.xml

```diff
@@ -1,18 +1,243 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
 <SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
-  <Enumeration>SummaryState_DisabledState,SummaryState_EnabledState,SummaryState_FaultState,SummaryState_OfflineState,SummaryState_StandbyState</Enumeration>
-  <Enumeration>DetailedState_DisabledState,DetailedState_EnabledState,DetailedState_FaultState,DetailedState_OfflineState,DetailedState_StandbyState</Enumeration>
+  <Enumeration>
+    DeviceId_chiller01P01=101,
+    DeviceId_chiller02P01=102,
+    DeviceId_chiller03P01=103,
+    DeviceId_crack01P02=201,
+    DeviceId_crack02P02=202,
+    DeviceId_fancoil01P02=301,
+    DeviceId_fancoil02P02=302,
+    DeviceId_fancoil03P02=303,
+    DeviceId_fancoil04P02=304,
+    DeviceId_fancoil05P02=305,
+    DeviceId_fancoil06P02=306,
+    DeviceId_fancoil07P02=307,
+    DeviceId_fancoil08P02=308,
+    DeviceId_fancoil09P02=309,
+    DeviceId_fancoil10P02=310,
+    DeviceId_fancoil11P02=311,
+    DeviceId_fancoil12P02=312,
+    DeviceId_manejadoraLower01P05=401,
+    DeviceId_manejadoraLower02P05=402,
+    DeviceId_manejadoraLower03P05=403,
+    DeviceId_manejadoraLower04P05=404,
+    DeviceId_manejadoraSblancaP04=501,
+    DeviceId_manejadoraSlimpiaP04=502,
+    DeviceId_vea01P05=601,
+    DeviceId_vea08P05=602,
+    DeviceId_vea09P05=603,
+    DeviceId_vea10P05=604,
+    DeviceId_vea11P05=605,
+    DeviceId_vea12P05=606,
+    DeviceId_vea13P05=607,
+    DeviceId_vea14P05=608,
+    DeviceId_vea15P05=609,
+    DeviceId_vea16P05=610,
+    DeviceId_vea17P05=611,
+    DeviceId_vea01P01=701,
+    DeviceId_vec01P01=702,
+    DeviceId_vin01P01=703,
+    DeviceId_vex03LowerP04=801,
+    DeviceId_vex04CargaP04=802
+  </Enumeration>
   <SALEvent>
     <Subsystem>HVAC</Subsystem>
-    <EFDB_Topic>HVAC_logevent_detailedState</EFDB_Topic>
+    <EFDB_Topic>HVAC_logevent_deviceEnabled</EFDB_Topic>
+    <Description>Event indicating that a device was enabled.</Description>
```

Comment:
"a device" implies this is for a single device (like the command).

Please consider "Report which devices are enabled" or similar.